core/transport/src/main/proto/proto.lock

```diff
@@ -3277,6 +3277,21 @@
               }
             ]
           },
+          {
+            "name": "S3SyntaxOptions",
+            "fields": [
```

Comment:
https://github.com/Alluxio/alluxio/pull/16651 added the missing proto.lock changes. We can cherry pick that PR to dora branch.